### PR TITLE
Feature/edit screen login

### PR DESCRIPTION
**Pull request by Tiago-C-Brandao**

### CODE REVIEW

File: pages/login.vue [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        alt=""
+        class="logo"
+      />
+      <form class="login">
+        <label>Username</label>
+        <input type="text">
```
Comment (ramirezmz) [inline pages/login.vue]: You can put class on input

File: pages/login.vue [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        <label>Password</label>
+        <input type="text">
```
Comment (ramirezmz) [inline pages/login.vue]: You can put class on input

File: pages/login.vue [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        <a href="">Esqueceu a senha?</a>
+        <button>LOGIN</button>
```
Comment (ramirezmz) [inline pages/login.vue]: You can put class here

File: pages/login.vue [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    </form>
+    </div>
+  </body>
+</template>
+
+<style>
+    * {
```
Comment (ramirezmz) [inline pages/login.vue]: Is this indentation correct?

**Review by ramirezmz — CHANGES_REQUESTED** [review]

File: pages/login.vue (hunk 1 above)
Comment (Tiago-C-Brandao) [reply pages/login.vue]: Create class on input

**Pull request merged**